scripts/lunchme.coffee

```diff
@@ -6,14 +6,24 @@
 #   hubot lunch me tacos - find what you are in the mood for
 #   hubot houston lunch me - find a place in Houston, too!
 
-radius = 5
+yelp = require "yelp"
+
+# Generate new keys at https://www.yelp.com/developers/manage_api_keys
+yelp = yelp.createClient
+  consumer_key: "uh4_dL5KECQPgl5OlS9AVg"
```

Comment:
I know we had the yelp api key here in plain view before, but what are your thoughts on pushing these 4 values into env variables that we can include in the bash script that sets them on the server?